test/topology/conftest.py

```diff
@@ -216,7 +216,9 @@ def fails_without_consistent_cluster_management(request, check_pre_consistent_cl
 # unless the cluster is dirty or the test has failed.
 @pytest.fixture(scope="function")
 async def random_tables(request, manager):
-    tables = RandomTables(request.node.name, manager, unique_name())
+    rf_marker = request.node.get_closest_marker("replication_factor")
+    replication_factor = rf_marker.args[0] if rf_marker is not None else 3  # Default 3
+    tables = RandomTables(request.node.name, manager, unique_name(), replication_factor)
```

Comment:
The default value for `replication_factor` in `RandomTables` is `1`, but there is only one use site of `RandomTables` and it's always passing `replication_factor` and the default is `3`.

Let's remove the default value inside `RandomTables` to make it less confusing.